src/common.rs

```diff
@@ -48,23 +48,66 @@ pub(crate) use crate::{
 
 // structs and enums
 pub(crate) use crate::{
-  alias::Alias, analyzer::Analyzer, assignment::Assignment,
```

Comment:
It's a big formatting change, but it's what I got from running the exact formatting commands you have in the GitHub Actions config file. 🤷‍♂️

Reply:
That's really strange, are you running `cargo +nightly fmt --all`?

What do `cargo +nightly --version` and `cargo +nightly fmt --version` show?

I get the following:

```
: cargo +nightly --version
cargo 1.44.0-nightly (ebda5065e 2020-04-16)
: cargo +nightly fmt --version
rustfmt 1.4.14-nightly (a5cb5d26 2020-04-14)
```

Reply:
Yup!

```plain
just on  issue_445 is 📦 v0.5.10 via 🦀 v1.42.0 
❯ cargo +nightly fmt --all

just on  issue_445 is 📦 v0.5.10 via 🦀 v1.42.0 
❯ cargo +nightly fmt --all -- --check

just on  issue_445 is 📦 v0.5.10 via 🦀 v1.42.0 
❯ echo $?
0

just on  issue_445 is 📦 v0.5.10 via 🦀 v1.42.0 
❯ git status
On branch issue_445
Your branch is up to date with 'origin/issue_445'.

nothing to commit, working tree clean
```

Reply:
Hmm, super weird, and how about the versions from `cargo +nightly --version` and `cargo +nightly fmt --version`? (I can see cargo  is at 1.42, but the nightly version should be different.)

Reply:
```plain
❯ cargo +nightly --version
cargo 1.44.0-nightly (ebda5065e 2020-04-16)

~ 
❯ cargo +nightly fmt --version
rustfmt 1.4.14-nightly (a5cb5d2 2020-04-14)
```

Reply:
Ah, okay, I figured it out, it's because of the braces in `justfile::{Justfile, Suggestion}`. When an item in a use block has braces, rustfmt switches to putting each item on its own line. You can put suggestion in its own module, i.e. `src/suggestion.rs`. I usually just put everything in its own module, since it's slightly easier to jump to.

Reply:
Ahhh, good find!